h/schemas/forms/admin/group.py

```diff
@@ -24,56 +22,65 @@
 VALID_GROUP_TYPES = (("restricted", _("Restricted")), ("open", _("Open")))
 
 
-@colander.deferred
-def group_creator_validator(node, kw):
-    def validate(form, value):
-        """
-        Validate that the creator username exists in the organization's authority.
+def username_validator(form, value):
+    """
+    Validate that the usernames exist in the organization's authority.
 
-        The creator of a group must belong to the same authority as the group
-        and the group's organization.  Validate that there is a user matching
-        the given creator username with the same authority as the chosen
-        organization.
+    The creator and members of a group must belong to the same authority as the
+    group and the group's organization.
+    """
+    user_svc = form.bindings["user_svc"]
 
-        """
-        user_svc = kw["user_svc"]
+    # Get the authority from the list of organisations based on the one
+    # actually picked in the form
+    authority = form.bindings["organizations"][value["organization"]].authority
```

Comment:
There was some highly over commented code here. I've packed this down to a single line

Reply:
I assure you that was me, even if I check history and it wasn't...seems like something I would do.